run.py

```diff
@@ -9,7 +9,7 @@
 
 
 def run(hand_tracker, model):
-    gestures = ["default", "left", "right", "select", "exit", "none"]
+    gestures = ["default", "left", "right", "select", "exit", "shortcut1", "shortcut2", "none"]
```

Comment:
Plz edit gestures in `HandTracker.py` file as well.

Reply:
edited